### PR TITLE
Adds support for TrustyAI Operator

### PR DESCRIPTION
**Pull request by alexcreasy**
Closes: #1276 

## Description

* Adds TrustyAI client capabilities for interacting with the operator and managing the service lifecycle. [non-visual]
* Adds Settings tab to project details page with a model bias section allowing for installing / uninstalling TrustyAI.
localhost_4010_projects_test1
localhost_4010_projects_test1 (1)
localhost_4010_projects_test1 (2)

## How Has This Been Tested?

Test 1: Verify TrustyAI CR creation and deletion
1) Create a new Data Science Project
2) Goto the Details -> Settings tab
3) Check the "Enable TrustyAI" checkbox
4) Verify you receive a success message and the checkbox is now ticked.
5) In the OpenShift console check that a `trustyai-service` CR has been installed in your namespace
6) Now check the Deployments tab for a trusty-service in your namespace.
7) Go back to the Dashboard and untick the checkbox
8) Verify a confirm modal appears, and confirm the delete
9) Verify the checkbox is unticked.
10) Go back to the OpenShift console and confirm the CR and service (from step 5 and 6) have been removed. 

Test 2: Verify TrustyAI works correctly
1) Go to Model Serving Global page
2) Open model `demo-loan-nn-onnx-alpha`
3) Verify the Model Bias tab appears.
4) Verify creating a Bias Configuration works
5) Verify the new configuration is selectable on the Charts view - NOTE: there will be no data for the charts as there is currently a bug in the TrustyAI service.
6) Verify Bias configuration duplicate and delete functionality works

Test 3: Verify Bias Tab is not visible when TrustyAI service is not installed
1) Goto project settings for `trustyai-e2e-modelmesh`
2) Uninstall the TrustyAI service.
3) Go back to the `demo-loan-nn-onnx-alpha` model serving screen
4) Verify the Model Bias tab is not visible

## Test Impact

TBD

## Request review criteria:

- [x] The commits have meaningful messages (squashes happen on merge by the bot).
- [x] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work
- [x] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

### CODE REVIEW

File: frontend/src/pages/modelServing/screens/metrics/MetricsPageTabs.tsx [hunk 1]
```diff
@@ -56,14 +60,15 @@ const MetricsPageTabs: React.FC = () => {
           <PerformanceTab />
         </Tab>
       )}
-      {biasMetricsEnabled && (
+      {biasMetricsEnabled && installed && (
```
Comment (DaoDaoNoCode) [inline frontend/src/pages/modelServing/screens/metrics/MetricsPageTabs.tsx:63]: Question here: can we check installed in `useBiasMetricsEnabled`? if it's not installed, the biasMetricsEnabled will always be false, right?

Comment (alexcreasy) [reply frontend/src/pages/modelServing/screens/metrics/MetricsPageTabs.tsx:63]: @DaoDaoNoCode I've just realised II forgot to hide the ModelBiasSettings section if the bias metrics feature flag is disabled. That means there will be no way to install it if the feature is disabled, so I don't think this should go in the feature flag hook.

I could create another hook for checking if it's enabled and installed? That would be a bit neater and make the logic reusable?

Comment (DaoDaoNoCode) [reply frontend/src/pages/modelServing/screens/metrics/MetricsPageTabs.tsx:63]: Yes fair point. I think just use another hook to wrap `biasMetricsEnabled` and `installed` would be fine. And on the project details settings you can just hide/show it based on the `biasMetricsEnabled` hook.

**Review by DaoDaoNoCode — APPROVED** [review]
lgtm!

**Pull request merged**